src/eXpansion/Bundle/AdminChat/ChatCommand/AdminShuffleCommand.php

```diff
@@ -83,7 +83,7 @@ public function execute($login, InputInterface $input)
             $maps[] = $map->fileName;
             $allMaps[] = $map->fileName;
             if (count($maps) > 250) {
-                $this->connection->removeMapList($maps,true);
+                $this->connection->removeMapList($maps);
```

Comment:
Ok but then the removeMapList for the ones that didn't fit the 200 can also be false, and you can remove the executeMulticall?